infra/values.yaml

```diff
@@ -0,0 +1,5 @@
+storage:
```

Comment:
Why this file? let vault generate it's own creds, and then we link the docs where we describe how to get the root credentials: https://jenkins-x.io/v3/admin/setup/secrets/vault/#using-the-vault-cli-directly

Reply:
Ok thanks

Reply:
Well actually the idea with adding a values file was that I wanted it to use in-memory storage and development mode, to make it more light weight. But I can definitely remove the devRootToken

Reply:
Vault seems to require 10G storage as default. https://github.com/hashicorp/vault-helm/blob/main/values.yaml#L590

Reply:
that makes sense, may be set the default storage to 1? I am not sure what is a good default?

Reply:
Ok, lets try that. I guess it makes it easier for people if vault remembers the secrets between runs.